src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleCard.tsx

```diff
@@ -12,22 +12,28 @@ import { ClimateSettingScheduleDeviceBar } from './ClimateSettingScheduleDeviceB
 
 interface ClimateSettingScheduleCardProps {
   climateSettingScheduleId: string
+  onSelectDevice: (deviceId: string) => void
```

Comment:
I wonder if this drilled prop should just be `onDeviceClick` everywhere instead of introducing a new name.

Reply:
I was following the example set by `AccessCodeDetails`

https://github.com/seamapi/react/blob/main/src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx#L103

Reply:
ok, lets follow that then